src/JustSaying/Properties/InternalsVisibleTo.cs

```diff
@@ -2,3 +2,4 @@
 
 [assembly: InternalsVisibleTo("JustSaying.IntegrationTests, PublicKey=00240000048000009400000006020000002400005253413100040000010001009d30c64bc42ba5037aec3cf60334df3a4c140ffc1dab19fdd31c9765e4e29afa441396ff9437a764b618c6473a3b350c56d706a0fbe83915f1f250ee23eb548b30306187ac685f65caa48303dc86f08c1c24d99dc84966273c207eaa8570b440004c7f49cef4ec77bc69118610ae2b53db7d8abeb465cbcd4bd190feaf517aad")]
 [assembly: InternalsVisibleTo("JustSaying.UnitTests, PublicKey=00240000048000009400000006020000002400005253413100040000010001009d30c64bc42ba5037aec3cf60334df3a4c140ffc1dab19fdd31c9765e4e29afa441396ff9437a764b618c6473a3b350c56d706a0fbe83915f1f250ee23eb548b30306187ac685f65caa48303dc86f08c1c24d99dc84966273c207eaa8570b440004c7f49cef4ec77bc69118610ae2b53db7d8abeb465cbcd4bd190feaf517aad")]
+[assembly: InternalsVisibleTo("JustSaying.Tools, PublicKey=00240000048000009400000006020000002400005253413100040000010001009d30c64bc42ba5037aec3cf60334df3a4c140ffc1dab19fdd31c9765e4e29afa441396ff9437a764b618c6473a3b350c56d706a0fbe83915f1f250ee23eb548b30306187ac685f65caa48303dc86f08c1c24d99dc84966273c207eaa8570b440004c7f49cef4ec77bc69118610ae2b53db7d8abeb465cbcd4bd190feaf517aad")]
```

Comment:
```suggestion
[assembly: InternalsVisibleTo("JustSaying.Tools, PublicKey=00240000048000009400000006020000002400005253413100040000010001009d30c64bc42ba5037aec3cf60334df3a4c140ffc1dab19fdd31c9765e4e29afa441396ff9437a764b618c6473a3b350c56d706a0fbe83915f1f250ee23eb548b30306187ac685f65caa48303dc86f08c1c24d99dc84966273c207eaa8570b440004c7f49cef4ec77bc69118610ae2b53db7d8abeb465cbcd4bd190feaf517aad")]
[assembly: InternalsVisibleTo("JustSaying.UnitTests, PublicKey=00240000048000009400000006020000002400005253413100040000010001009d30c64bc42ba5037aec3cf60334df3a4c140ffc1dab19fdd31c9765e4e29afa441396ff9437a764b618c6473a3b350c56d706a0fbe83915f1f250ee23eb548b30306187ac685f65caa48303dc86f08c1c24d99dc84966273c207eaa8570b440004c7f49cef4ec77bc69118610ae2b53db7d8abeb465cbcd4bd190feaf517aad")]
```

Reply:
I was actually trying to put them in alphabetical order, but forgot how to alphabet 😆